CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added|Changed|Deprecated|Removed|Fixed|Security
 Nothing so far
 
+## 5.1.4 - 2021-11-15
```

Comment:
Ik zou em 5.2.0 dopen. PHP 8 support is een nieuwe feature

Reply:
Was nu de consensus dat dit een nieuwe major versie ging worden? Dan zou het hier nog even moeten worden aangepast

Reply:
Nee, hier is nog geen consensus over. Geldt voor de PR's in alle andere bundles ook.